### ISSUE TITLE
Lack of examples

### ISSUE THREAD

**Issue by uhidontkno** [ciscoheat/sveltekit-rate-limiter#9]
Examples should exist for ratelimiting on `+server.ts` and `hooks.server.ts`

**Comment by ciscoheat** [comment]
Good idea, feel free to contribute to OSS by adding them to the readme.

**Comment by uhidontkno** [comment]
> Good idea, feel free to contribute to OSS by adding them to the readme.
I can't contribute because I was looking for an API route example for this library because I do not know this library and would like to use an example. That's why I created an issue instead of a pull request.
